src/assets/src/components/queue.tsx

```diff
@@ -412,14 +412,21 @@ const ChangeMeetingTypeDialog = (props: ChangeMeetingTypeDialogProps) => {
                 <Modal.Title>Change Meeting Type</Modal.Title>
             </Modal.Header>
             <Modal.Body>
-                <div className="row col-lg">
-                    <p>Select Meeting Type</p>
-                    <p className="required">*</p>
-                </div>
-                <BackendSelector backends={props.backends}
-                    allowedBackends={new Set(props.queue.allowed_backends)}
-                    onChange={props.onChangeBackend}
-                    selectedBackend={props.selectedBackend}/>
+                <Row>
```

Comment:
Do you think the col-lg on line 372 might also be an issue and need to be updated?

Reply:
It's not an issue per se, but it's an instance of the direct Bootstrap class usage I've tried to move away from. Actually, this column turned out to be not needed, since it's not part of a larger grid structure (other `Row`s or `Col`s). Since this was the last instance of direct Bootstrap class usage in the module, I went ahead and removed it and used an `Alert` component there. It does have one side effect of setting `role="alert"`, which makes it pretty noisy for screenreaders, but we're using `Alert` for other usages on the page. I could see setting `role={undefined}` for the SMS and management alerts, but they are conditional as well. Going to leave it like this for now.